portal-frontend/src/app/features/create-application-dialog/create-application-dialog.component.spec.ts

```diff
@@ -4,14 +4,15 @@ import { ApplicationService } from '../../services/application/application.servi
 import { CodeService } from '../../services/code/code.service';
 
 import { CreateApplicationDialogComponent } from './create-application-dialog.component';
+import { SharedModule } from '../../shared/shared.module';
 
 describe('CreateApplicationDialogComponent', () => {
   let component: CreateApplicationDialogComponent;
   let fixture: ComponentFixture<CreateApplicationDialogComponent>;
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      imports: [MatDialogModule],
+      imports: [MatDialogModule, SharedModule],
```

Comment:
This seems like a code smell, if its a unit test why do you need to import the SharedModule?

Reply:
For mat radio group, I'll add it explicitly

Reply:
You should use schemas: [NO_ERROR_SCHEMA] to prevent errors from rendering